content/en/security/application_security/setup/php/compatibility.md

```diff
@@ -0,0 +1,143 @@
+---
+title: PHP Compatibility Requirements
+aliases:
+  - /security/application_security/threats/setup/compatibility/php
```

Comment:
@estringana this alias exists on the `content/en/security/application_security/setup/compatibility/php.md` page as well. which pages should this alias belong to?

Reply:
Think there's some misunderstanding of alias on our team, what do they do? they should appear once and only once?

We have quite a few pages already merged where several pages share the same alias.

Reply:
okay, I removed it from the other file.